include/tvm/relax/op_attr_types.h

```diff
@@ -49,6 +49,14 @@ using FInferStructInfo =
  */
 using FCallPacked = String;
 
+/*!
+ * \brief Legalizes an expression with another expression. This function will be
+ *  invoked in Legalize pass. It is a target-dependent pass.
+ * \param bb The blockbuilder context.
+ * \param call The original call.
+ */
+using FRelaxLegalize = runtime::TypedPackedFunc<Expr(const BlockBuilder& bb, const Call& call)>;
```

Comment:
No need for the `Relax` prefix I think. Other attributes don’t have it as well. 
```suggestion
using FLegalize = runtime::TypedPackedFunc<Expr(const BlockBuilder& bb, const Call& call)>;
```